requirements.txt

```diff
@@ -1,7 +1,7 @@
 scipy>=1.3.3
 altair>=4.0.0
 numpy>=1.16.5
-pandas>=1.2.0,<1.3.0
+pandas
```

Comment:
> *I've also updated the requirements to require `pandas>= 1.3.0`*

There seems to be no minimum bound here?

Reply:
I agree that enforcing the pandas>=1.3.0 would be useful here, we originally took this out because we didn't want to force users into updating their pandas version from 1.2 to 1.3 because of their updates to Lux. 

@westernguy2, @cgarciae : Would love to get your thoughts on this! With this PR fix, my understanding is that we no longer support pandas<1.3? Does using pandas 1.2 cause errors in Lux or fail silently?

Reply:
The PR fix does now support the most up to date version of `pandas` and `pandas<1.3`. In `lux/core/__init__`, I check for the version, so the correct variable assignment is completed based on the version. 

`pandas 1.2` should work as expected

Reply:
Got it, thanks @westernguy2 !